build.gradle.kts

```diff
@@ -28,4 +29,8 @@ allprojects {
         maven { url = URI("https://kotlin.bintray.com/kotlinx") }
         maven { url = URI("https://dl.bintray.com/kotlin/ktor") }
     }
+}
+
+tasks.withType<Test> {
+    maxParallelForks = Runtime.getRuntime().availableProcessors().minus(1).coerceAtLeast(1)
```

Comment:
Why that? 🤔

Reply:
This allows to run the test suite concurrently, using all processors (minus one) on your machine

Reply:
Oh right, nice!